explorer/info.go

```diff
@@ -0,0 +1,158 @@
+package explorer
+
+import (
+	"go.sia.tech/core/consensus"
+	"go.sia.tech/core/types"
+)
+
+// ChainStats contains a bunch of statistics about the consensus set as they
+// were at a specific block.
+type ChainStats struct {
+	Block             types.Block
+	ValidationContext consensus.ValidationContext
+
+	// Transaction type counts.
+	SpentSiacoinsCount         uint64
+	SpentSiafundsCount         uint64
+	NewSiacoinsCount           uint64
+	NewSiafundsCount           uint64
+	NewFileContractsCount      uint64
+	RevisedFileContractsCount  uint64
+	ResolvedFileContractsCount uint64
```

Comment:
It's unclear to me whether these counts are very useful in practice. Let's remove them for now.